integration_tests/test_erc20.py

```
@@ -0,0 +1,72 @@
+        ],
+        gas=300000,
+    )
+    cli = mantra.cosmos_cli()
+    cli.query_erc20_token_pairs()
+    # deposit should be nop
+    before = await w3.eth.get_balance(owner)
+    weth = WETH(to=WETH_ADDRESS)
+    before = await balance_of(w3, ZERO_ADDRESS, owner)
+    receipt = await weth.fns.deposit().transact(w3, account, value=1000)
+    fee = receipt["effectiveGasPrice"] * receipt["gasUsed"]
+    await balance_of(w3, WETH_ADDRESS, owner) == 1000
```

Ruff (0.12.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)